### DISCUSSION TITLE
Amazing News feed For you!

### DISCUSSION THREAD

**Discussion by Horizon733** [community/community#13225] [Other Feature Feedback, Questions, & Ideas] [answered]
`For you` section is good, would love it if more repository suggestions are added based on the personal repo, fork, starred repos instead of follow recommendations. Other than that, everything is really good.

**Comment by tiziodcaio** [comment]
Hm maybe add the possibility to delete it there are people who don't like it

**Comment by sntwandy** [comment]
They could combine several ways to provide various information within the feed.

**Reply by H3l1opy** [reply]
Eles podiam facilitar a navegaçao pra quem esta aprendendo a programa eu nao sei muito de programação to estudando e ainda sou brasileiro tenho que traduzir todas as paginas pra português isso dificulta minha esperiencia

**Comment by arfon** [comment, marked as answer]
Thanks for the positive feedback @Horizon733 and you suggestions on how we could improve the recommendations.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
